src/components/TypeTester/TypeTester.js

```diff
@@ -338,9 +338,8 @@ export default class TypeTester extends Component {
               min={100}
               max={1600}
               value={this.state.typeSizeMultiplier}
-              key={this.state.typeSizeMultiplier}
```

Comment:
@vpicone don't know why, but this line of code is what was breaking the slider. It did not like having a key 👀

Reply:
@jnm2377 that was a hacky work around against an upstream Carbon issue: https://github.com/carbon-design-system/carbon/issues/6039

Without it, nothing happened at all. Now that we've updated Carbon removing this was the right move!